CHANGELOG.md

```diff
@@ -28,6 +29,54 @@
 - [v0.5.0](#v050---2020-08-06)
 <!-- END TOC -->
 
+-------------------------------------------------
+## v0.26.0 - 2022-09-19
+
+### Release notes
+
+- In 1.7 the cert-manager API versions v1alpha2, v1alpha3, and v1beta1, have been removed from the custom resource definitions (CRDs).
+- In 1.8 the cert-manager will validate the spec.privateKey.rotationPolicy on Certificate resources. Valid options are Never and Always.
```

Comment:
Now I sound like a broken record but these should not be here, as they were in the previous release.